Dockerfile

```
@@ -2,142 +1,127 @@
+WORKDIR /srv/app
 
-RUN xcaddy build \
-	--with github.com/dunglas/mercure \
-	--with github.com/dunglas/mercure/caddy \
-	--with github.com/dunglas/vulcain \
-	--with github.com/dunglas/vulcain/caddy
+COPY --from=php_extension_installer --link /usr/bin/install-php-extensions /usr/local/bin/
 
-# Prod image
-FROM php:8.3-fpm-alpine3.20 AS app_php
+RUN apt-get update && apt-get install --no-install-recommends -y \
```

Hadolint (2.12.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)